data/fixtures/test_data.py

```diff
@@ -556,7 +557,7 @@ def __init__(self):
         db.session.commit()
 
     def add_course(self, num_students=5, num_assignments=1, num_additional_criteria=0, num_groups=0, num_answers='#',
-            with_comments=False, with_draft_student=False, with_comparisons=False, with_self_eval=False):
+            with_comments=False, with_draft_student=False, with_comparisons=False, with_self_eval=False, num_non_comparable_ans=0):
```

Comment:
have `num_non_comparable_ans` set to at least 1 by default (so there is better test coverage overall)